xmake/core/sandbox/modules/import/core/sandbox/sandbox.lua

```diff
@@ -28,6 +28,7 @@ local sandbox_core_sandbox = sandbox_core_sandbox or {}
 -- load modules
 local sandbox   = require("sandbox/sandbox")
 local raise     = require("sandbox/modules/raise")
+local history   = require("project/history")
```

Comment:
This project history module will generate `.xmake` directory in the current directory, but `xmake lua` is a global action. It does not depend on project.
 
I think it would be better to save repl history in the global directory. .e.g `~/.xmake/cache/history_repl`